src/tests/index.test.js

```diff
@@ -59,7 +58,6 @@ pluginTester({
       pluginOptions: {
         namespace: undefined,
       },
-      snapshot: false,
```

Comment:
Removing `snapshot: false` for these tests causes them to fail on my machine (a Mac):

```
Invariant Violation: Code was unmodified but attempted to take a snapshot. If the code should not be modified, set `snapshot: false`
```

All tests pass if the `snapshot: false` params are restored.

Reply:
I didn't get what would be used to compare both results if not the snapshot

Reply:
Shouldn't we provide a `outputFixtures` at least?

Reply:
Oh, I think I got it. I'm trying to solve.

Reply:
One of the reasons seems to be line breaking chars in the comparison , **probably** It relates to [this function](https://github.com/babel-utils/babel-plugin-tester/blob/master/src/plugin-tester.js#L227) within the babel-plugin-tester.
This lib is 4 major versions behind the babel-plugin-tester (we are at v5.5.2 and they are at 9.2.0), should we try to update it? If so I think it would be nice to create another pull request for updating the lib.